app/src/main/java/in/ac/iitb/gymkhana/iitbapp/MainActivity.java

```diff
@@ -47,6 +52,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
         NavigationView navigationView = (NavigationView) findViewById(R.id.nav_view);
         navigationView.setNavigationItemSelectedListener(this);
+        if (ContextCompat.checkSelfPermission(this, android.Manifest.permission.ACCESS_FINE_LOCATION) !=
```

Comment:
Then After asking for permission the button doesnt appear unless u reopen the fragment.. How can that be done?